test_regress/t/t_trace_cat.cpp

```diff
@@ -26,8 +27,8 @@ int main(int argc, char** argv, char** env) {
     Verilated::debug(0);
     Verilated::traceEverOn(true);
 
-    VerilatedVcdC* tfp = new VerilatedVcdC;
-    top->trace(tfp, 99);
+    std::unique_ptr<VerilatedVcdC> tfp{new VerilatedVcdC};
+    top->trace(tfp.get(), 99);
```

Comment:
Can you use this fix in driver.pl and trace_public_func instead?  Much cleaner, thanks.  Feel free to merge then.

Reply:
35248659b3b8ed1805984e92b9b4c7f56f4a1788 uses unique_ptr in those cases.
 will merge after CI passes.